src/components/employeeDetails.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <br>
+  <h1 class="text-center">UNIVERSAL MARKETING CO., LTD</h1>
```

Comment:
This was updated in previous merges to have the title in the App. Vue. We can take those off.

Reply:
Will update and remove title.